src/WebJobs.Script.Grpc/Server/RetryProxyHandler.cs

```diff
@@ -41,6 +42,19 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
 
                     await Task.Delay(currentDelay, cancellationToken);
                 }
+                catch (Exception ex)
+                {
+                    if (attemptCount == _maxRetries)
+                    {
+                        _logger.LogWarning("Reached the maximum retry count for worker request proxying. Error: {exception}", ex);
+                    }
+                    else
+                    {
+                        _logger.LogWarning($"Unsupported exception type in {nameof(RetryProxyHandler)}. Request will not be retried. Exception: {{exception}}", ex);
```

Comment:
Was the double `{{}}` intentional here, or should it be a single `{}` for the exception?

Reply:
yeah I don't think we need the double `{{}}` here. Let me push a change for this

Reply:
just kidding we do need the double braces. Otherwise we run into this error: `azure-functions-host/src/WebJobs.Script.Grpc/Server/RetryProxyHandler.cs(53,146): error CS0103: The name 'exception' does not exist in the current context [/workspaces/azure-functions-host/src/WebJobs.Script.Grpc/WebJobs.Script.Grpc.csproj]`